MDA/MolSSI workshop page with registration (#379)
MDAnalysis/MDAnalysis.github.io: _posts/2024-05-14-ASUworkshop.md

This two-day workshop is intended to transform attendees from users to
developers and will cover an introduction to MDAnalysis, software best
practices, and guide participants through the creation of their own [MDAKit][mdakits].

### Workshop outline

MDAnalysis developers and experienced MolSSI instructors will teach
participants how to build their own MDAnalysis-based Python packages by guiding
them through the development of their own MDAKits. Specifically, the workshop
will include 3 modules: (1) an introduction to using and writing custom
analyses in MDAnalysis; (2) an overview of software development and maintenance
best practices; and (3) an interactive hackathon session where participants
write their own MDAKits.

The program will run from the morning of Monday, June 24th until the early
evening of Tuesday June 25th. 

Parts of the workshop will be held in a hybrid mode for those unable to attend
in-person. Registration for the hybrid components is required. Recordings of
hybrid sessions will be made available after the workshop.

Breakfast and lunch will will be provided together with tea/coffee during
breaks.

### Registration

> Attendance at this workshop will be *free*. 

Reviewer: Do we need to state the number of individuals the in-person meeting will be limited to?

Author: I added 40 in-person.

I don't know how the hybrid portion will work, that's more @jennaswa 's domain.

Reviewer 2: Based on our experience with the hybrid TYC workshop, it was very manageable to have 1-2 folks answering questions from participants online (we had ~20 people). Everyone online kept their videos off and didn't take us up on offers to unmute and ask questions, so we can probably accommodate at least another 40 online. I would say we don't have to limit it now, and can make a decision based on the number of applications we receive, which sessions we decide to make hybrid, and how many instructors we have able to help with the hybrid portion during each session.

accommodation* (single room within 5 minutes of the workshop venue) will be
awarded.

#### Application with bursary request (non-local attendees)

Register by **May 27, 2024** (midnight anywhere on Earth).

We especially welcome your application if you consider yourself being part of
an under-represented demographic in the computational molecular sciences.

Bursaries are reserved to enable the attendance of participants from outside
the greater Phoenix area.

Note that airfare for travel awardees will have to be booked by June 3 through
our travel agency so as an awardee you will have to be able to respond
immediately and act quickly.

#### Application without bursary request or local attendees

Register by **June 5, 2024** (midnight anywhere on Earth).

### Travel

[Tempe][] is part of the "Valley of the Sun" metropolitan area, which encompasses
[Phoenix][], the 5th most populous city in the US. Today the area is home to more
than 4.8 million people and encompasses the ancient lands of the Akimel O'odham
(Pima), Maricopa, and Tohono O'odham tribes.

#### Airport

**Phoenix Sky Harbor** [PHX][] airport has direct connections to all major hubs
in the US.

> PHX airport is 5.5 miles (10 minutes by car/rideshar) from the workshop venue.

Reviewer 3: ```suggestion
PHX airport is 5.5 miles (10 minutes by car/rideshare) from the workshop venue.
```

#### Car/rideshare

To get from the airport to Tempe either take a cab (traditional or self-driving
Waymo) or rideshare (Lyft, Uber).

> However, public transport works quite well, too.

Reviewer 3: I'm not used to the US, but shouldn't public transports be the first suggestion/choice?

Reviewer: > I'm not used to the US

Europe has entered the chat :)

I do agree that if public transport is safe & effective it should be listed first - although I also know that this isn't always the case everywhere.

Author: Generally I agree with you and I re-arranged.

Public transport in Phoenix is not good _except_ when going along the Light Rail. @ianmkenney does have some interesting storied about encounters on the Light Rail at night but generally I'd consider it reasonably safe. If your destination is close to the Light Rail then use it, otherwise I'd take a car.

Reviewer 3: That's fair. If they are not particularly safe or effective, then it's fine to list alternative first.

them through the development of their own MDAKits. Specifically, the workshop
will include 3 modules: (1) an introduction to using and writing custom
analyses in MDAnalysis; (2) an overview of software development and maintenance
best practices; and (3) an interactive hackathon session where participants
write their own MDAKits.

The program will run from the morning of Monday, June 24th until the early
evening of Tuesday June 25th. 

Parts of the workshop will be held in a hybrid mode for those unable to attend
in-person. Registration for the hybrid components is required. Recordings of
hybrid sessions will be made available after the workshop.

Breakfast and lunch will will be provided together with tea/coffee during
breaks.

### Registration

Attendance at this workshop will be *free* but limited to 40 in-person
participants.

A number of **bursaries** will be available to assist with travel and
accommodation costs for participants from outside the Phoenix area. Bursaries
(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
accommodation* (single room within 5 minutes of the workshop venue) will be
awarded.

> 

Author: Do we want to make a single registration button here?

Author: I added one that looks like this

![image](https://github.com/MDAnalysis/MDAnalysis.github.io/assets/237980/dda4e7fa-1a8f-4bd7-b22a-b3d7932a3817)

#### Application *with bursary request* (non-local attendees)

Register by **May 27, 2024** (midnight anywhere on Earth).

We especially welcome your application if you consider yourself being part of
an under-represented demographic in the computational molecular sciences.

Bursaries are reserved to enable the attendance of participants from outside
the greater Phoenix area.

Note that airfare for travel awardees will have to be *booked by June 3 through
our travel agency* so as an awardee you will have to be able to
respond to our communications (via email) immediately and act quickly.

> Bursary recipients will be notified by May 29. 

Author: @IAlibay @fiona-naughton @jennaswa @armcdona we need to tell anyone who applied for a bursary asap so I added a quick turnaround for us.

Does this work?

I know that this only leaves a day for reviewing but it will be tight either way and there's not much we can do about it. 5/27 is a Thu, 5/28 is Fri.

Author: On May 29 I would want to tell any bursary applicants 
* if they got a bursary (travel + hotel) or just hotel,
* if they have been accepted (even if no bursary)

In the application form we should ask *(1) Will you attend only if you get travel + hotel? (2) Will you attend if you get hotel? (3) Will you attend without additional support?*

If we only informed bursary recipients that they have been accepted and not the category (3) (will attend regardless) then we make it difficult for them to book travel at reasonable cost.

Reviewer 2: The registration form currently asks about attendance based on financial support in a slightly different way. After asking if people need support for hotel and/or travel, it asks estimated total costs (for travel) as well as amount of $ that could be obtained elsewhere (e.g., university travel awards, PI grants, etc.). It asks generally if they'd be unable to attend without the support asked for, so I think we could make decisions to answer the above.

Happy to change it to the way described above (or feel free to do so at https://docs.google.com/forms/d/1-_iVRym3oOxMkozB2TnBPWVUdk6ALJpAVt2fpNV9s8M/edit), but was just worried about people having fatigue answering so many questions

Author: The reg form looks good, thanks @jennaswa . I agree with you that it provides the information we need.

#### Application without bursary request or local attendees

> Register by **June 5, 2024** (midnight anywhere on Earth).

Author: We want to tell non-local applicants ASAP if they have been accepted so that they can plan travel (see above). If we wait until June 5+X to tell them, they have less than 2 weeks. The alternative is to tell _all_ non-locals to apply for a bursary, regardless of if they need it, which will add even more to our reviewing load.

We could say that application deadline for non-locals is May 27. However, then we effectively give priority to non-locals as we don't know how many locals want to come.

Therefore, I suggest that we **keep a single deadline for all applications** and we make this deadline **May 27 for everyone**. In this way we will have all information when we need it.

*If we don't get enough applications we can then extend it until June 5*.

Opinions @IAlibay @fiona-naughton @jennaswa @armcdona  ?

Reviewer 2: I agree that we should just keep it simple and have one deadline.

---
layout: post
title: "MDAnalysis/MolSSI Workshop at Arizona State University (June 24/25, 2024)"
---

<img
src="/public/images/ASUworkshop_cactus.png"
title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 24-25th, 2024 at Arizona State University"
style="float: right; height: 20em; " />

<img
src="/public/images/MolSSI_Logo.png"
title="MolSSI logo" alt="MolSSI logo"
style="float: right; height: 5em; " />

MDAnalysis, in conjunction with the [Molecular Sciences Software Institute][MolSSI]

> (MolSSI) and with the support of the [Chan-Zuckerberg Inititative][CZI] and the

Reviewer 2: ```suggestion
(MolSSI) and with the support of the [Chan Zuckerberg Initiative][CZI] and the
```

> [Center for Biological Physics][CBP], is holding a **2-day** workshop on **June

Reviewer 2: ```suggestion
[Center for Biological Physics][CBP], is holding a **free, 2-day workshop** on **June
```

24th-25th** 2024 at **Arizona State University** in Tempe, Arizona, USA.

This two-day workshop is intended to transform attendees from users to
developers and will cover an introduction to MDAnalysis, software best
practices, and guide participants through the creation of their own [MDAKit][mdakits].

### Workshop outline

MDAnalysis developers and experienced MolSSI instructors will teach
participants how to build their own MDAnalysis-based Python packages by guiding
them through the development of their own MDAKits. Specifically, the workshop
will include 3 modules: (1) an introduction to using and writing custom
analyses in MDAnalysis; (2) an overview of software development and maintenance
best practices; and (3) an interactive hackathon session where participants
write their own MDAKits.

The program will run from the morning of Monday, June 24th until the early
evening of Tuesday June 25th. 

Parts of the workshop will be held in a hybrid mode for those unable to attend
in-person. Registration for the hybrid components is required. Recordings of
hybrid sessions will be made available after the workshop.

> Breakfast and lunch will will be provided together with tea/coffee during

Reviewer 2: ```suggestion
```

> breaks.

Reviewer 2: ```suggestion
```

### Registration

Attendance at this workshop will be *free* but limited to 40 in-person
participants.

A number of **bursaries** will be available to assist with travel and

> accommodation costs for participants from outside the Phoenix area. Bursaries

Reviewer 2: ```suggestion
accommodation costs for participants from outside the [greater Phoenix area](https://en.wikipedia.org/wiki/Phoenix_metropolitan_area). Bursaries
```

(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
accommodation* (single room within 5 minutes of the workshop venue) will be
awarded.

> <a href="https://LINK.TO.REGISTRATIONFORM" target="_blank" style="background:#FF9200;padding:10px;margin:10px 0px;text-align:center;text-decoration:none;font-size:12pt;color:#000000;display:inline-block;border-radius:3px">Register</a>

Reviewer 2: ```suggestion
<a href="https://forms.gle/UeiFJ6rbnXnhosnWA" target="_blank" style="background:#FF9200;padding:10px;margin:10px 0px;text-align:center;text-decoration:none;font-size:12pt;color:#000000;display:inline-block;border-radius:3px">Register</a>
```

#### Application *with bursary request* (non-local attendees)

Register by **May 27, 2024** (midnight anywhere on Earth).

We especially welcome your application if you consider yourself being part of

> an under-represented demographic in the computational molecular sciences.

Reviewer 2: ```suggestion
an underrepresented demographic in the computational molecular sciences.
```

Bursaries are reserved to enable the attendance of participants from outside

> the greater Phoenix area.

Reviewer 2: ```suggestion
the [greater Phoenix area](https://en.wikipedia.org/wiki/Phoenix_metropolitan_area).
```

Note that airfare for travel awardees will have to be *booked by June 3 through

> our travel agency* so as an awardee you will have to be able to

Reviewer 2: ```suggestion
our travel agency*, so as an awardee you will have to be able to
```

respond to our communications (via email) immediately and act quickly.

> Bursary recipients will be notified by May 29. 

Reviewer 2: ```suggestion
Bursary recipients will be notified by May 30. 
```
The timeline works for me (though as a note, May 27 is a Mon, and May 28 is a Tues). I am only changing the date since we say _by_ May XX. That way, we have May 28 to review/decide, and can send notifications out _on_ May 29.

Author: I appreciated the detail to semantics. 

I left it as "on May 29 or earlier" to convey that we are keeping it tight. I'd like people to start looking out for notification starting May 29. 

The reg form also contains May 29.

5. The stop closest to the Hyatt Place Hotel and the workshop venue is
**University Dr/Rural Rd** (Stop #10023). If you want to get off closer to
downtown Tempe get off one stop earlier at **Veterans Way/College Ave** (Stop #10025)
6. Walk to your destination.

#### Car/rideshare

To get from the airport to anywhere in Tempe either take a cab (traditional or self-driving
Waymo) or rideshare (Lyft, Uber).

#### Workshop Venue

The workshop will be held in the Student Success Center in the Department of
Physics at [Arizona State University][ASU] (room PSF 186 in the [Physical Sciences F
building][PSF]). The address is

  Department of Physics  
  Arizona State University  
  Physical Sciences F Building  
  550 E Tyler Mall, PSF 186  
  Tempe, AZ 85287  
  USA
 
Paid parking is available in the [Tyler Street Parking
Structure][TylerStParking] and other on-campus parking structures.

### Accommodation

The closest hotel (5 min by foot) is the [Hyatt Place Tempe][HyattPlace].

Other hotels are available in Tempe within walking distance.

### Weather

Arizona is *hot* during June. Temperatures in excess of 110ºF (43.3ºC) are not

> rare so it is important that you bring a hat for sun protection and carry

Reviewer 2: ```suggestion
rare, so it is important that you bring a hat for sun protection and carry
```

enough water with you.

> Avoid hiking in the desert during the hot portion of the day (10am - 6pm) as it

Reviewer 2: ```suggestion
Avoid hiking in the desert during the hot portion of the day (10am - 6pm), as it
```

can be deadly.

> 

Reviewer 2: ### Who to Contact

If you have any questions or special requests related to this workshop, you may [contact the organizing committee](mailto:workshops@mdanalysis.org).
